.github/workflows/pypi-test.yml

```diff
@@ -27,7 +27,6 @@ permissions: read-all
 jobs:
   pypi-test:
     name: Test PyPI Distribution
-    if: ${{ github.event.workflow_run.conclusion != 'failure' }}
     runs-on: ${{ matrix.os }}
```

Comment:
revert please (and commit with `git commit --amend --no-edit` to ensure things stay squashed)